packages/commons-ui-next/NextComposed.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+import NextLink from "next/link";
+import PropTypes from "prop-types";
+import React from "react";
+
+const NextComposed = React.forwardRef(function NextComposed(props, ref) {
+  const { as, href, locale, prefetch, ...other } = props;
+
+  return (
+    <NextLink as={as} href={href} locale={locale} prefetch={prefetch}>
```

Comment:
`NextLink` supports a whole lot more props than this. See [docs](https://nextjs.org/docs/api-reference/next/link) and MUI example [integration](https://github.com/mui/material-ui/blob/master/examples/nextjs/src/Link.js).